README.md

```diff
@@ -48,96 +48,120 @@ your Cloud SQL instance.
 The instance connection name for your Cloud SQL instance is always in the
 format "project:region:instance".
```

Comment:
```suggestion
format `project:region:instance`. It can also be found on the [Cloud SQL Instance Overview page](https://console.cloud.google.com/sql/instances).
```